sources/sort.c

```diff
@@ -124,7 +124,7 @@ VOID WriteStats(POSITION *plspace, WORD par, WORD checkLogType)
 		 * LogHandle. This must be done within the ErrorMessageLock region to
 		 * avoid a data race between threads. */
 		const WORD oldLogHandle = AC.LogHandle;
-		if ( checkLogType && AM.LogType ) {
+		if ( checkLogType == CHECKLOGTYPE && AM.LogType ) {
```

Comment:
Well, I understand why you added `== CHECKLOGTYPE`...; it's logically the right way, but typically, this generates a longer binary code because a register has to be compared with `1` (though it should be completely negligible, of course). 

([Compiler Explorer example](https://gcc.godbolt.org/z/d9TvcdYon): function `foo1` has `cmp edi,0x1` (`83 ff 01`) while function `foo2` has `test edi,edi` (`85 ff`))

At this point, we don't need to add `== CHECKLOGTYPE`, I think. If later we need to extend `checkLogType` to ternary or more, we can change it at that time.

Reply:
OK, I will drop this again in the squash.

Reply:
Can you also resolve the conflict with https://github.com/vermaseren/form/commit/3ccda3a3145036a7a8def3ae3bf7b3dbe8066646?

Reply:
Done in principle... I actually don't understand why github considers what remains, a conflict.

Reply:
In this case, I think you need to rebase the upstream master branch (`vermaseren/master`):
```bash
git switch issue-470
git fetch upstream  # I assume it points to vermaseren/master
git reset --hard upstream/master
git cherry-pick 0b0443e
# here the conflict should be manually resolved
git add -u
git cherry-pick --continue
```
Then
```bash
git range-diff HEAD^..HEAD 0b0443e^..0b0443e
```
should print only the resolved conflict.

If you make a mistake or have a trouble in these steps, then you can always reset the branch to `upstream/master` or `0b0443e`.